ci/psn.yaml

```diff
@@ -0,0 +1,164 @@
+---
+unpack_release: &unpack_release
+  platform: linux
+  params:
+    CLUSTER_PUBLIC_KEY:
+  run:
+    path: /bin/bash
+    args:
+    - -euo
+    - pipefail
+    - -c
+    - |
+      echo "preparing keyring to verify release..."
+      echo "${CLUSTER_PUBLIC_KEY}" > key
+      gpg --import key
+      gpg --verify gsp/source.tar.gz.asc
+      echo "unpacking src tarball..."
+      tar -xvf gsp/source.tar.gz -C platform --strip-components=1
+  inputs:
+  - name: gsp
+  outputs:
+  - name: platform
+
+resource_types:
+- name: terraform
+  type: registry-image
+  source:
+    repository: govsvc/terraform-resource
+    tag: 0.13.0-beta.2
+- name: github
+  type: registry-image
+  source:
+    repository: ((github-resource-image))
+    tag: ((github-resource-tag))
+- name: concourse-pipeline
+  type: docker-image
+  source:
+    repository: concourse/concourse-pipeline-resource
+    tag: "2.2.0"
+
+resources:
+- name: gsp
+  type: github-release
+  source:
+    owner: alphagov
+    repository: gsp
+    access_token: ((github-api-token))
+    release: true
+    tag_filter: gsp-v([^v].*)
+
+- name: users
+  type: github-release
+  source:
+    owner: alphagov
+    repository: gds-trusted-developers
+    access_token: ((github-api-token))
+    release: true
+
+- name: src
+  type: github
+  source:
+    uri: https://github.com/alphagov/verify-cluster-config.git
+    branch: master
+    organization: alphagov
+    repository: verify-cluster-config
+    github_api_token: ((github-api-token))
+    approvers: ((github-approvers))
+    required_approval_count: 2
+    commit_verification_keys: ((trusted-developer-keys))
+
+- name: psn-state
+  type: terraform
+  source:
+    env_name: ((account-name))
+    backend_type: s3
+    backend_config:
+      bucket: cd-gsp-private-qndvvc
+      region: eu-west-2
+      key: psn-cluster-((cluster-name)).tfstate
+    vars:
+      aws_account_role_arn: ((account-role-arn))
+      gsp_cluster_state_bucket_name: cd-gsp-private-qndvvc
+      gsp_cluster_state_bucket_key: cluster-((cluster-name)).tfstate
+      workspace_name: ((account-name))
+      vpc_endpoint: ((psn-vpc-endpoint))
```

Comment:
where does this come from?

Reply:
It's a secret that Concourse pulls from SSM, added via hijacking the `info` job in the GSP Concourse team and following the instructions to add a new secret.